tasks/geographica/ca/csv.py

```diff
@@ -0,0 +1,107 @@
+import os
+import json
+import csv
+
+from luigi import Task, WrapperTask, Parameter
+from luigi.local_target import LocalTarget
+from tasks.ca.statcan.data import AllCensusTopics, AllNHSTopics
+from tasks.meta import current_session
+from lib.logger import get_logger
+
+LOGGER = get_logger(__name__)
+
+# TODO Add block level when we have ACS for block
+GEOGRAPHY_LEVELS = {
+    'GEO_PR': 'ca.statcan.geo.pr_',
+    'GEO_CD': 'ca.statcan.geo.cd_',
+    'GEO_DA': 'ca.statcan.geo.da_'
+}
+
+
+class Measurements2CSV(Task):
```

Comment:
This task and the one for AU should be very similar. Could we refactor this so all the `Measurements2CSV` tasks inherit from a base task?

Reply:
Yes, and the US too. I want to finish UK and then refactor them